datacats/docker.py

```diff
@@ -103,14 +103,15 @@ def _get_docker():
 
 class WebCommandError(Exception):
 
-    def __init__(self, command, container_id, logs):
+    def __init__(self, command, container_id, logs, ssh=False):
```

Comment:
Good point. Do you mind if I address this? I will just go through all the occurances manually.